pkgs/top-level/python-packages.nix

```diff
@@ -4453,7 +4453,7 @@ in {
 
     propagatedBuildInputs = with self; [
       six keystoneclient prettytable oslo-utils oslo-i18n lxml httplib2 cliff
-      dogpile_cache appdirs anyjson pbr openstackclient
+      dogpile_cache appdirs anyjson pbr pkgs.openstackclient
```

Comment:
This is going to cause trouble. What will happen is that all the `propagatedBuildInputs` of `openstackclient` will also be added to the `PYTHONPATH` of this package. Is your intention that it should have `openstackclient` on `PATH`?